### ISSUE TITLE
GeneratedPluginRegistrant.swift changes formatting depending on platform

### ISSUE THREAD

**Issue by mtcliatt** [google/flutter-desktop-embedding#600]
FDE tree-ish: 4340dec
Flutter tree-ish: fa4d31b31

Mac:
```
import Foundation
import FlutterMacOS
```

Linux:
```
import FlutterMacOS
import Foundation
```

**Comment by krisgiesing** [comment]
Huh, why is Linux even generating/formatting this file?

**Comment by stuartmorgan** [comment]
What you have labelled Mac was not generated from treeish fa4d31b31. That part of the template is [a static string](https://github.com/flutter/flutter/blob/fa4d31b31/packages/flutter_tools/lib/src/plugins.dart#L484) so it's impossible for the order to be different. That output is from a tree that is at least a month older than fa4d31b31, before [this change](https://github.com/flutter/flutter/pull/40195).

> Huh, why is Linux even generating/formatting this file?

Generated registrants are updated for all platforms supported by a project at the same time.

**Comment by mtcliatt** [comment]
Please re-open

**Comment by mtcliatt** [comment]
Here we are: https://paste.googleplex.com/6354012355952640

**Comment by stuartmorgan** [comment]
The diff at the end of that paste is showing that what was checked in was the old, pre-linked-above fix, and that building on macOS at fa4d31b31 is, as expected, updating it to the new, correctly sorted version.

```
--- a/[...]/macos/Flutter/GeneratedPluginRegistrant.swift
+++ b/[...]/macos/Flutter/GeneratedPluginRegistrant.swift
@@ -1,8 +1,9 @@
 //
 //  Generated file. Do not edit.
 //
-import Foundation
+
 import FlutterMacOS
+import Foundation
```
